### ISSUE TITLE
v2: version check 

### ISSUE THREAD

**Issue by FFroehlich** [PEtab-dev/libpetab-python#349]
`pteab.v2.Problem.from_yaml` will only accept "2.0.0" as version string, even though it sounds like "2" would also be valid according to spec.

see https://github.com/PEtab-dev/libpetab-python/blob/99373c4340512def5a23bf2a75805c3f88251251/petab/v2/problem.py#L139

**Comment by dweindl** [comment]
Here it says we use SemVer: https://petab.readthedocs.io/en/latest/development.html#versioning-of-the-petab-format

Here it says integer: https://petab.readthedocs.io/en/latest/v1/documentation_data_format.html#yaml-file-for-grouping-files

Needs to be clarified. I wouldn't see a problem with implicit zeros for minor/patch.

**Comment by FFroehlich** [comment]
> Here it says we use SemVer: https://petab.readthedocs.io/en/latest/development.html#versioning-of-the-petab-format
> 
> Here it says integer: https://petab.readthedocs.io/en/latest/v1/documentation_data_format.html#yaml-file-for-grouping-files
> 
> Needs to be clarified. I wouldn't see a problem with implicit zeros for minor/patch.

Here it says string/pep440 https://petab.readthedocs.io/en/latest/v2/documentation_data_format.html#yaml-file-for-grouping-files

**Comment by dweindl** [comment]
> Here it says string/pep440 https://petab.readthedocs.io/en/latest/v2/documentation_data_format.html#yaml-file-for-grouping-files which I thought implies implicit zeros with minor/patch, but that‘s also not super evident.

True. That needs to be simplified. I think we want to drop the epoch there, and keep any PEP references out of it.

I will fix the initial issue.

**Comment by dweindl** [comment]
It seems this was all addressed here and in the v2 specs. Reopen if not.
